.github/workflows/datafuse-performance-dispatch.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       PR_NUMBER: ${{ github.event.client_payload.PR_NUMBER }}
       LAST_COMMIT_SHA: ${{ github.event.client_payload.LAST_COMMIT_SHA }}
       UUID: ${{ github.event.client_payload.UUID }}
-      ITERATION: 10
+      ITERATION: 100
```

Comment:
LGTM, just wondering if we have a way to specify how many times it runs.

Reply:
During the deisgn of test infra, I thought it could be add to /run-perf as a variable payload, but that maybe too complex for developers to use it

Reply:
Ok, keep it easy